compiler/rustc_codegen_ssa/src/mir/debuginfo.rs

```diff
@@ -366,6 +366,24 @@ impl<'a, 'tcx, Bx: BuilderMethods<'a, 'tcx>> FunctionCx<'a, 'tcx, Bx> {
                     return;
                 }
 
+                // LLVM doesn't handle stores on some of the internal SVE types that we are required
```

Comment:
It looks like this is target independent file but we are adding target specific code. I'm not sure this belongs here. I would prefer to generalize this as such:

```
if target.some_property(operand)
  return
```

I am also confused why we're only returning in the case that we have a `<vscale x i1 x 16>` type. Why that type specifically? What about a `<vscale x i1 x 32>` or a `<vscale x i32 x 4>`?

On RISC-V we can spill all vector types. It is done using a vector store instruction. I am pretty sure SVE can do this too.